src/qlever/commands/example_queries.py

```diff
@@ -431,6 +573,16 @@ def execute(self, args) -> bool:
                         result_size = run_command(
                             f"sed 1d {result_file}", return_output=True
                         )
+                    elif accept_header == "application/qlever-results+json":
+                        try:
+                            # sed cmd to get the number between 2nd and 3rd double_quotes
+                            result_size = run_command(
+                                f"jq '.res[0]' {result_file}"
+                                " | sed 's/[^0-9]*\\([0-9]*\\).*/\\1/'",
+                                return_output=True,
+                            )
```

Comment:
Shouldn't there be one function for extracting the single count from the result if there is one? Right now there is special code for this spread over several places